hydradx/model/plot_utils.py

```diff
@@ -1,224 +1,27 @@
-import matplotlib.pyplot as plt
+import matplotlib.pyplot as plot
```

Comment:
matplotlib.pyplot as plt is industry standard, let's stick with it

Reply:
k